CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+**Breaking Changes**:
```

Comment:
Since this is a large fundamental change of how Relay operates in Proxy mode it's a good idea to call that out.

Ideally the PR description then goes into details why this is a breaking change and/or what users are expected to do when they encounter this one.